package.json

```diff
@@ -66,6 +68,7 @@
   },
   "scripts": {
     "build": "webpack --config webpack.conf.js",
+    "build:types": "tsc --emitDeclarationOnly && api-extractor run && rm -rf dist/typings",
     "prebuild": "del-cli dist",
     "lint": "eslint 'src/**/*.ts'",
     "prepare": "npm run build",
```

Comment:
I think we need to make it `"prepare": "npm run build && npm run build:types"`

Reply:
Right, missed that one